File: SDKs/Aspose.Slides_Cloud_SDK_for_Java/src/main/java/com/aspose/slides/model/OrderedMergeRequest.java
package com.aspose.slides.model;

import java.util.*;
public class OrderedMergeRequest {
  private List<PresentationToMerge> Presentations = new ArrayList<PresentationToMerge>();
  /**
	 * getPresentations
	 * Gets List<PresentationToMerge>
	 * @return Presentations
	 */
  public List<PresentationToMerge> getPresentations() {
    return Presentations;
  }

	/**
	 * setPresentations
	 * Sets List<PresentationToMerge>
	 * @param Presentations List<PresentationToMerge>
	 */
  public void setPresentations(List<PresentationToMerge> Presentations) {
    this.Presentations = Presentations;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class OrderedMergeRequest {\n");
    sb.append("  Presentations: ").append(Presentations).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Slides_Cloud_SDK_for_CSharp/src/Com/Aspose/Slides/Model/OrderedMergeRequest.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Slides.Model {
  public class OrderedMergeRequest {
    public List<PresentationToMerge> Presentations { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class OrderedMergeRequest {\n");
      sb.Append("  Presentations: ").Append(Presentations).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
